### DISCUSSION TITLE
how to visualize skeleton animation from mixamo fbx?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [google/filament#5656] [General]
how to visualize skeleton animation from mixamo fbx?

**Comment by prideout** [comment]
I've never used Mixamo.  Filament is just a renderer, but we do provide with an asset loader for glTF.  We can help with specific questions about our glTF loader posted to the discussions tab.

**Comment by bejado** [comment]
@jinfagang FYI you can import a mixamo fbx into Blender, then output a glTF file which should be compatible with our gltfio loader library.

**Comment by lucasjinreal** [comment]
@bejado Hi, thanks for the info.

I saw that filament using assimp as model file reader, why not add fbx directly support?

**Reply by bejado** [reply]
We use assimp in a very limited capacity (mainly for internal samples) and don't support all the features of FBX, namely animation.
